### ISSUE TITLE
N-Acetylaspartic acid levels and ASPA relation

### ISSUE THREAD

**Issue by zafush** [obophenotype/human-phenotype-ontology#4233]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Abnormality of N-Acetylaspartic acid levels in urine 

Extra Notes: (Blood and CSF can also be added seperately under Abnormality of N-Acetylaspartic acid levels)

### 2. Synonyms
Abnormality of N-acetyl-L-aspartic acid levels in urine (blood and CSF)
Abnormality of N-acetylaspartate levels in urine (blood and CSF)
Abnormality of N-acetyl aspartate levels in urine (blood and CSF) 
Abnormality of N-acetyl aspartic acid levels in urine (blood and CSF)
Abnormality of NAA levels in urine (blood and CSF)

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
Elevated N-acetylaspartic acid (NAA) in urine using gas chromatography-mass spectrometry.

Other notes: Although NAA concentration is also elevated in the blood and cerebrospinal fluid (CSF) of children with neonatal/infantile (severe) Canavan disease, elevated concentration of NAA in urine is sufficient for diagnosis of affected individuals.
References: (1) Michals K, Matalon R. Canavan disease. In: Raymond GV, Eichler F, Fatemi A, Naidu S, eds. Leukodystrophies. London: Mac Keith Press; 2011:156-69, (2) PMID: 8412017

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser]
http://compbio.charite.de/hpoweb/showterm?id=HP:0010899

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
OMIM 271900

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
ORCID-id: 0000-0002-2085-5773

NOTE: Annotations also misses HP:0025052 (detection of NAA by MRS) and ASPA gene (Canavan  Disease).

**Comment by pnrobinson** [comment]
Thanks for the suggestion. I have two comments.
There is no condition with "Abnormally low N-acetylaspartic acid urine concentration" and so I think we should make a term with the label "Elevated N-acetylaspartic acid urine level"
The HPO has a separate branch for urinary lab abnormalities, and so I will put the term there.

**Comment by pnrobinson** [comment]
I am adding these terms as "Increased" only since there is no medically relevant "Decreased" etc.
